src/view/com/modals/ContentFilteringSettings.tsx

```diff
@@ -48,15 +48,19 @@ export const Component = observer(({}: {}) => {
       <ScrollView style={styles.scrollContainer}>
         <View style={s.mb10}>
           {isIOS ? (
-            <Text type="md" style={pal.textLight}>
-              Adult content can only be enabled via the Web at{' '}
-              <TextLink
-                style={pal.link}
-                href="https://bsky.app"
-                text="bsky.app"
-              />
-              .
-            </Text>
+            store.preferences.adultContentEnabled ? (
+              <></>
```

Comment:
nit: this can be `null` to stick with convention of the rest of the codebase

Reply:
ah yep